test/TooltipSpec.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Created by madmax on 10.01.17.
+ */
+/* global describe, it, expect */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import Tooltip from '../src/Tooltip';
+
+describe('<Tooltip />', () => {
+  it('Renders correctly if there is only one child node', () => {
+    shallow(<Tooltip tooltip='123'><div /></Tooltip>);
+  });
+
+  it('Renders child node with class tooltipped', () => {
+    const wrapper = shallow(<Tooltip tooltip='123'><div /></Tooltip>);
```

Comment:
since all these wrappers are the same you could add a `beforeEach` at the beginning and remove all these